js/client/modules/@arangodb/testsuites/rta_makedata.js

```diff
@@ -109,6 +109,7 @@ function makeDataWrapper (options) {
         count += 1;
         let args = pu.makeArgs.arangosh(this.options);
         args['server.endpoint'] = this.instanceManager.findEndpoint();
+        args['log.file'] = fs.join(fs.getTempPath(), `rta_out_${count}.log`);
```

Comment:
does this (new) logfile need to be garbage-collected at test end?

Reply:
its created in the test-directory of this testsuite, and hence will (not) be collected along with the instance logs on (non-)success.

Reply:
fixed.